samples/modules/chre/sample.yaml

```diff
@@ -18,7 +18,7 @@ tests:
         - "D: Instance ID 1 assigned to app ID 0x0000000000000001.*"
         - "EchoApp::nanoappStart\\(\\)"
         - "EchoApp::nanoappHandleEvent\\(sender_instance_id=0, event_type=1,
-          event_data@\\(nil\\)\\)"
+          event_data@.*\\)"
```

Comment:
what about making this (nil|0) or something like that?

Reply:
sure, would be a better test that way. I've updated the branch.